whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/RelationType.java

```diff
@@ -0,0 +1,18 @@
+package net.ripe.db.whois.api.rdap.domain;
+
+public enum RelationType {
+    UP("up"),
+    TOP("top"),
+    DOWN("down"),
+    BOTTOM("bottom");
+
+    private final String value;
+
+    RelationType(String value) {
+        this.value = value;
```

Comment:
How are invalid relation types handled?

Reply:
Add an integration test for an "invalid" relation type. Expect 400 Bad Request and "unknown relation" in the response.

Reply:
Thanks, I missed this test I will update https://github.com/RIPE-NCC/whois/pull/1603 with it